conu/apidefs/container.py

```diff
@@ -177,7 +200,8 @@ def start(self):
         """
         raise NotImplementedError("start method is not implemented")
 
-    def exec(self, command, **kwargs):
+    # exec is keyword in python
+    def exec_command(self, command, **kwargs):
```

Comment:
it is question if we should use execute, or exec_command. I have no preference, but maybe closed to __execute__ instead of exec_command, it is too long

Reply:
I don't personally have a strong preference, honestly -- I agree with you, the shorter, the better. What do others think? @hodovska @phracek @bkabrda

Reply:
will change to execute if no one responds by tomorrow

Reply:
I am also fine with execute